Assignment/Noura Aldablan/NouraS.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+<meta charest="utf-8">
+  <title>Noura's Page</title>
+  <link rel="stylesheet" href="css/NouraS.css">
+
+  </head>
+
+  <body>
+
+    <header>
+
+      <ul>
+        <li><a href="file:///C:/Users/DELL/Desktop/Noura%20Aldablan/NouraD.html"><i>About me</i></a> </li>
```

Comment:
Well done @Aldablan. I recommended you to use the relative path not the absolute. because I can run it easy on my computer.